tests/NHSD.BuyingCatalogue.Ordering.Api.UnitTests/Controllers/OrdersControllerTests.cs

```diff
@@ -460,6 +460,50 @@ async Task<ActionResult<CreateOrderResponseModel>> CreateOrder()
             Assert.ThrowsAsync<ArgumentNullException>(CreateOrder);
         }
 
+        [Test]
+        public async Task DeleteOrderAsync_DeleteOrderSuccessful_ReturnsNoContent()
+        {
+            var context = OrdersControllerTestContext.Setup();
+
+            var response = await context.OrdersController.DeleteOrderAsync("Order");
+            response.Should().BeOfType<NoContentResult>();
+            context.Order.IsDeleted.Should().BeTrue();
```

Comment:
Would also like to suggest splitting these assertions out as well, similar to a previous comment.